tools/test_live_repair.sh

```diff
@@ -91,63 +108,44 @@ if ! ps -p $dsc_pid > /dev/null; then
 fi
 
 gen=1
-# Initial seed for verify file
+# Seed the initial volume
+echo "$(date) Begin pretest initial fill" | tee -a "$test_log"
 if ! "$crucible_test" fill --dsc 127.0.0.1:9998 -q -g "$gen"\
-          --verify-out "$verify_log" >> "$test_log" 2>&1 ; then
-    echo Failed on initial verify seed, check "$test_log"
+          --skip-verify >> "$test_log" 2>&1 ; then
```

Comment:
Is this just to go faster?

Reply:
The addition of `--skip-verify`, yeah, that's just to go a little faster.
We don't bother with the `--verify-out` any longer because we are not stopping and starting crutest several times, so we no longer need to read in the verify info from the previous loop.

The stopping/starting part of the test here was causing problems when we have multiple region sets.  The region set that we ended the previous test loop on might not match up with the region set that we start with by default.